### ISSUE TITLE
GithubPullRequestFormatter raises exception when pronto-rails_schema runner has a warning

### ISSUE THREAD

**Issue by dennmart** [mmozuras/pronto#175]
I recently started using `GithubPullRequestFormatter` with Pronto and ran into an issue where when the [pronto-rails_schema](https://github.com/raimondasv/pronto-rails_schema) runner has a warning. This is the relevant stack trace from the Rake task I use:

```
undefined method `new_lineno' for nil:NilClass excluded from capture due to environment or should_capture callback
rake aborted!
NoMethodError: undefined method `new_lineno' for nil:NilClass
/home/ubuntu/fino/vendor/bundle/ruby/2.3.0/gems/pronto-0.7.1/lib/pronto/formatter/github_pull_request_formatter.rb:13:in `line_number'
/home/ubuntu/fino/vendor/bundle/ruby/2.3.0/gems/pronto-0.7.1/lib/pronto/formatter/git_formatter.rb:69:in `new_comment'
/home/ubuntu/fino/vendor/bundle/ruby/2.3.0/gems/pronto-0.7.1/lib/pronto/formatter/git_formatter.rb:77:in `block in new_comments'
/home/ubuntu/fino/vendor/bundle/ruby/2.3.0/gems/pronto-0.7.1/lib/pronto/formatter/git_formatter.rb:77:in `map'
/home/ubuntu/fino/vendor/bundle/ruby/2.3.0/gems/pronto-0.7.1/lib/pronto/formatter/git_formatter.rb:77:in `new_comments'
/home/ubuntu/fino/vendor/bundle/ruby/2.3.0/gems/pronto-0.7.1/lib/pronto/formatter/git_formatter.rb:8:in `format'
/home/ubuntu/fino/vendor/bundle/ruby/2.3.0/gems/pronto-0.7.1/lib/pronto.rb:59:in `block in run'
/home/ubuntu/fino/vendor/bundle/ruby/2.3.0/gems/pronto-0.7.1/lib/pronto.rb:58:in `each'
/home/ubuntu/fino/vendor/bundle/ruby/2.3.0/gems/pronto-0.7.1/lib/pronto.rb:58:in `run'
```

Other runners I have work properly with `GithubPullRequestFormatter`, and pronto-rails_schema works well with other formatters.

I'm not sure if it's an issue with the runner or with Pronto itself. I found a similar issue in this repo (#78), so maybe it's something among the same lines (no line number is passed to Pronto [as seen here](https://github.com/raimondasv/pronto-rails_schema/blob/master/lib/pronto/rails_schema.rb#L39-L42)).

**Comment by mmozuras** [comment]
@dennmart this was fixed in [v0.8.1](https://github.com/mmozuras/pronto/releases/tag/v0.8.1). Please reopen if you'll encounter it again 😄.

/cc @raimondasv (author of [pronto-rails_schema](https://github.com/raimondasv/pronto-rails_schema).

**Comment by dennmart** [comment]
@mmozuras Thank you so much!
